.changeset/cyan-rivers-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
```

Comment:
Note: remember to add the missing changesets for the major (I will write a migration guide in a separate PR)